index.bs

```diff
@@ -2439,6 +2439,10 @@ Council Composition</h5>
 	Membership of each [=Council=] instance is fixed at formation,
 	and is not changed by any [=AB=] or [=TAG=] elections
 	occurring before that Council has reached a conclusion.
+	Specifically, the list of potential [=Council=] members evolves
+	as terms start or end
```

Comment:
```suggestion
	The list of potential [=Council=] members evolves
	as terms start and end
```

Reply:
Ok with swapping or for and, but why remove specifically? My intent was to indicate that this is a clarification of the previous sentence, rather than an additional requirement. I guess I'd be ok with turning the period at the end of the previous sentence instead of saying Sepcifically at the start of the next one, but having neither weakens the link between the two statements.